packages/ui/src/components/EventsCard/EventsCard.test.tsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+
+import { EventsCard } from './EventsCard';
+import { BUTTON_TEXT, events as listOfEvents, MAIN_TITLE } from './EventsCard.mocks';
+
+describe('EventsCard tests', () => {
+  describe('render component tests', () => {
+    it('main title should be render', () => {
+      // when
+      render(<EventsCard events={listOfEvents} />);
+
+      // then
+      const title = screen.getByText(MAIN_TITLE);
+
+      expect(title).toBeInTheDocument();
+    });
+
+    describe('render events tests', () => {
+      it('should be render 3 events when events arg contain 3 events', () => {
+        // when
+        render(<EventsCard events={listOfEvents} />);
+
+        // then
+        const allEvents = screen.getAllByTestId('event');
+
+        expect(allEvents.length).toEqual(listOfEvents.length);
+      });
+
+      it('buttons should be render with text Przejdź do wydarzenia', () => {
+        // when
+        render(<EventsCard events={listOfEvents} />);
+
+        // then
+        const allButtons = screen.getAllByRole('button');
+
+        (allButtons || []).map((button) => expect(button).toHaveTextContent(BUTTON_TEXT));
+      });
+    });
+  });
+
+  describe('date tests', () => {
+    it('date should be render like 01.11 when date arg is new Date(2012, 10, 1, 10, 10)', () => {
+      // given
+      const events = [
+        {
+          id: 0,
```

Comment:
💭 You can create a `createEventMock` function in the `EventsCard.mocks.ts` file. It will take the `overrides` param to customize particular mocks. This way you can reuse it in all places you create an event mock.